README.md

```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
```

Comment:
- [ ] As requested by the previous reviewers, please kindly consider adding code linters correctly for this repository, **Code linters** help to diagnose and fix technical issues—e.g. code smells—in the code.
Please kindly visit the html-css in the [Linters Config](https://github.com/microverseinc/linters-config/tree/master/html-css) 😊 and follow the steps closely.

- [ ] 🟢 Check linters:
Linter result should be green ✔️ . AKA: Linter should report NO errors.
There should be no custom changes to linter config (as an attempt to cheat linter error reports).

- [ ] If you need help on how to setup linters correctly, kindly ask for help on the slack Chanel **"microverse full-time stundent"** and you will surely get someone to help. 😊